test/consistency.jl

```diff
@@ -84,6 +84,9 @@ function consistent_functions(nlps; rtol=1.0e-8, exclude=[])
     Hs = Vector{Any}(undef, N)
     for i = 1:N
       (I, J, V) = hess_coord(nlps[i], x)
+      IS, JS = hess_structure(nlps[i])
+      @test IS == I
+      @test JS == J
```

Comment:
Should this be `.==` or `all(...)`?

Reply:
`IS == I` is valid syntax.

Reply:
You're right. It seems to contradict the dot notation in other places. I always do `all(IS .== I)`.